src/Analyzers/Core/CodeFixes/FileHeaders/AbstractFileHeaderCodeFixProvider.cs

```diff
@@ -195,7 +208,7 @@ private SyntaxTriviaList CreateNewHeader(string prefixWithLeadingSpaces, string
         {
             var copyrightText = GetCopyrightText(prefixWithLeadingSpaces, expectedFileHeader, newLineText);
             var newHeader = copyrightText;
-            return SyntaxFacts.ParseLeadingTrivia(newHeader);
```

Comment:
Why? `ISyntaxFacts.ParseLeadingTrivia` is available in both the shared analyzer and code fixes layer.

Reply:
Yep missed this one. Now fixed.